CONTRIBUTING.md

```diff
@@ -148,8 +148,7 @@ fetch_latest_backup('data/')
 
 
 ##### Suppliers information (CNPJ)
-1. `src/fetch_cnpj_info.py` iterates over the CEAP datasets looking for supplier unique documents (CNPJ) and creates a local dataset with each supplier info.
-1. `src/clean_cnpj_info_dataset.py` clean up and translate the supplier info dataset.
```

Comment:
Thanks @Irio ! I did the corrections you mentioned, but I think would be nice to wait for @martini97. 

He suggested a improvement in the script described [here](https://github.com/datasciencebr/serenata-de-amor/pull/221#issuecomment-298145149) so when @martini97 finish it we can merge this PR. What do you think?